test/CardsTests.cpp

```diff
@@ -4,35 +4,18 @@
 
 struct CardsTests : public ::testing::Test
 {
-    json dataJson = R"({
-                    "0" : "Zobowiązany jesteś zmodernizować swoje miasto, płacisz za każdy dom 80 $,za każdy hotel 230 $",
-                    "1" : "Remontujesz swoje domy. Płacisz do banku za każdy dom 50 $, za każdy hotel 200 $.",
-                    "2" : "Wracasz do Madrytu.",
-                    "3" : "Mandat za szybko jazdę. Płacisz 30 $.",
-                    "4" : "Cofasz się o trzy pola.",
-                    "5" : "Wracasz na start.",
-                    "6" : "Rozwiązałeś dobrze krzyżówkę. Jako 1 nagrodę otrzymujesz 200 $.",
-                    "7" : "Idziesz do więzienia. Nie przechodzisz przez start. Nie otrzymujesz premii 400 $.",
-                    "8" : "Płacisz opłatę za szkołę 300 $.",
-                    "9" : "Idziesz do Neapolu. Jeżeli przechodzisz przez start otrzymujesz 400 $.",
-                    "10": "Piłeś w czasie pracy, płacisz karę 40 $.",
-                    "11": "Wracasz do Brukseli. Jeżeli przechodzi przez start otrzymujesz 400 $.",
-                    "12": "Bank wypłaca Ci procenty w wysokości 100 $.",
-                    "13": "Idziesz do Kolei Wschodnich. Jeżeli przechodzisz przez start otrzymujesz 400 $.",
-                    "14": "Bank wpłaca Ci należne odsetki w wysokości 300 $.",
-                    "15": "Wychodzisz wolny z więzienia. Kartę należy zachować do wykorzystania lub sprzedania."
-                })"_json;
-    Cards redCards{"red", dataJson};
+    json dataJson = R"({"2" : "Wracasz do Madrytu."})"_json;
```

Comment:
You can extract a string "Wracasz do Madrytu" to a const and use it here and in line 23 to avoid duplications.